Prac_5/word_occurrences.py

```diff
@@ -0,0 +1,13 @@
+word_ot_count = {}
+text_input = input("Enter a string: ")
+words = text_input.split()
+
+for word in words:
+    count = word_ot_count.get(word, 0)
+    word_ot_count[word] = count + 1
+
+words = list(word_ot_count.keys())
+
+longest_word = max(len(word) for word in words)
```

Comment:
This is a great way of finding the longest word. Wish i had thought of that instead of using those bulky for loops :)